src/Core/Components/Checkbox/FluentCheckbox.razor.cs

```diff
@@ -10,6 +11,16 @@ public partial class FluentCheckbox : FluentInputBase<bool>
     private bool? _checkState = false;
     private const string JAVASCRIPT_FILE = "./_content/Microsoft.FluentUI.AspNetCore.Components/Components/Checkbox/FluentCheckbox.razor.js";
 
+    protected override string? ClassValue
+    {
+        get
+        {
+            return new CssBuilder(base.ClassValue)
+                .AddClass("checked", Value)
```

Comment:
Could you add the name of the "when" parameter to be clearer about the meaning of "value"?
`.AddClass("checked", when: Value)`